visualization/app/codeCharta/ui/blacklistPanel/blacklistPanel.component.html

```diff
@@ -48,7 +48,7 @@ <h3 class="header-text">Excluded</h3>
 		</md-list-item>
 
 		<md-list-item
-			ng-repeat="item in $ctrl._viewModel.exclude | orderBy: ['path'] track by $ctrl._viewModel.exclude.indexOf(item)"
+			ng-repeat="item in $ctrl._viewModel.exclude | orderBy: ['path']"
```

Comment:
```suggestion
			ng-repeat="item in $ctrl._viewModel.exclude | orderBy: ['path'] track by item.path"
```

Reply:
I willl look into it and give you a feedback.